controllers/releaseplanadmission/suite_test.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2022.
```

Comment:
```suggestion
Copyright 2023.
```
![15fba009c3cc837d-600x338](https://github.com/redhat-appstudio/release-service/assets/1498203/b7634887-b55a-49fc-bc46-df34e7383c7b)